pages/guides/transfer-between-users/create-verified-customer.mdx

```diff
@@ -12,7 +12,7 @@ meta:
 
```

Comment:
All of the step numbers in the headers are off from the step numbers in the title for this guide